src/phase-manager.ts

```diff
@@ -73,15 +239,26 @@ interface PokemonFaintInit {
  * This is responsible for managing the game's {@linkcode Phase | phases}.
  */
 export class PhaseManager {
-  /** dequeue/remove the first element to get the next phase */
+  /**
+   * The main queue where {@linkcode Phase | Phases} are scheduled.
+   * The first Phase in this queue is {@linkcode Phase.start | run} whenever
+   * {@linkcode shiftPhase} is called.
+   */
   private phaseQueue: Phase[] = [];
-  /** A temporary storage of what will be added to the front of {@linkcode phaseQueue} */
+  /**
+   * When {@linkcode shiftPhase} is called, the Phases in this queue are inserted
+   * in queue order to the front of {@linkcode phaseQueue}.
+   */
   private phaseQueuePrepend: Phase[] = [];
   /** overrides default of inserting phases to end of phaseQueuePrepend array, useful for inserting Phases "out of order" */
   private phaseQueuePrependSpliceIndex: number = -1;
   private conditionalQueue: [() => boolean, Phase][] = [];
```

Comment:
```suggestion
  /** @deprecated see {@link https://github.com/Despair-Games/poketernity/pull/910#discussion_r2029764830} */
  private conditionalQueue: [() => boolean, Phase][] = [];
```
Also add to `pushConditionalQueue`.